Config.sample.toml

```diff
@@ -521,7 +521,7 @@ DATA_LANG_NAME = "ja-Hrkt"
 BACKPACKER = "バックパッカー"
```

Comment:
Should we delete the backpacker phrases now that we don't use it?

Reply:
Indeed !